packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -107,7 +107,7 @@ function Tooltip(props) {
   const handleOwnRef = React.useCallback(ref => {
     setChildNode(ReactDOM.findDOMNode(ref));
   }, []);
-  const handleRef = useForkRef(childNode, handleOwnRef);
+  const handleRef = useForkRef(children.ref, handleOwnRef);
```

Comment:
Great catch.

Reply:
haha 👍, we have 300k users that can do it for us too 😛.